client/src/components/forms/MoodForm.js

```diff
@@ -1,7 +1,274 @@
+
+  const meditationTimeInput = meditation => {
+    if (meditation === true) {
+      return (
+        <FormControl className={classes.formControl}>
+          <InputLabel id="meditation-time-attribute">Meditation Time</InputLabel>
+          <Select
+            labelId="meditation-time-attribute"
+            id="meditation-time-attribute"
+            value={meditationTime}
+            onChange={handleMeditationTimeChange}
+          >
+            <MenuItem value={10}>10 min</MenuItem>
+            <MenuItem value={20}>20 min</MenuItem>
+            <MenuItem value={30}>30 min</MenuItem>
+            <MenuItem value={40}>40 min</MenuItem>
+            <MenuItem value={50}>50 min</MenuItem>
+            <MenuItem value={60}>60 min</MenuItem>
+          </Select>
+        </FormControl>
+      );
+    } else {
+    }
+  };
+
+  const useStyles = makeStyles(theme => ({
+    root: {
+      width: 300 + theme.spacing(3) * 2
+    },
+    margin: {
+      height: theme.spacing(3)
+    }
+  }));
+
+  const MoodSlider = withStyles({
+    root: {
+      color: "#52af77",
+      height: 8
+    },
+    thumb: {
+      height: 24,
+      width: 24,
+      backgroundColor: "#fff",
+      border: "2px solid currentColor",
+      marginTop: -8,
+      marginLeft: -12,
+      "&:focus,&:hover,&$active": {
+        boxShadow: "inherit"
+      }
+    },
+    active: {},
+    valueLabel: {
+      left: "calc(-50% + 4px)"
+    },
+    track: {
+      height: 8,
+      borderRadius: 4
+    },
+    rail: {
+      height: 8,
+      borderRadius: 4
+    }
+  })(Slider);
+  const handleSleepChange = event => {
+    setSleep(event.target.value);
+  };
+
+  const handleMedChange = event => {
+    setMed(event.target.value);
+  };
+
+  const handleExerciseChange = event => {
+    setExercise(event.target.value);
+  };
+
+  const handleExerciseTimeChange = event => {
+    setExerciseTime(event.target.value);
+  };
+
+   const handleYogaChange = event => {
+    setYoga(event.target.value);
+  };
+
+  const handleYogaTimeChange = event => {
+    setYogaTime(event.target.value);
+  };
+
+  const handleMeditationChange = event => {
+    setMeditation(event.target.value);
+  };
+
+  const handleMeditationTimeChange = event => {
+    setMeditationTime(event.target.value);
+  };
+
+  const classes = useStyles();
+
+  return (
+    <Container>
+      <div className={classes.root}>
+        <Typography gutterBottom>Mood</Typography>
+        <MoodSlider
+          valueLabelDisplay="on"
+          aria-label="pretto slider"
+          defaultValue={5}
+          min={0}
+          max={10}
+        />
+        <FormGroup>
+          <FormControl className={classes.formControl}>
+            <InputLabel id="sleep-attribute">Sleep</InputLabel>
+            <Select
+              labelId="sleep-attribute"
+              id="sleep-attribute"
+              value={sleep}
+              onChange={handleSleepChange}
+            >
+              <MenuItem value={1}>1 hr</MenuItem>
```

Comment:
These and the other selects can be generated programmatically like:
```
{Array(24).fill(null).map(_, x => <MenuItem value={x+1}>{x + 1} hrs</MenuItem>)}
```
With a slight variation on the ones that don't increment by one. 
This creates an array `Array(24)` and in JavaScript it won't let you map over an array if it's values are empty, so we call `.fill(null)` to give us an array we can map over, we then map over said array using the index value (the variable x) and use that to know which value to generate for the MenuItem